src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((_, reject) => {
+  setTimeout(() => {
+    reject();
+  }, 3000);
+});
+
+const onSuccess = () => {
+  const successMessage = document.createElement('div');
+  successMessage.className = 'message';
+  successMessage.textContent = 'Promise was resolved!';
+  document.body.appendChild(successMessage);
+};
+
+const onError = () => {
+  const errorMessage = document.createElement('div');
+  errorMessage.className = 'message error-message';
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.appendChild(errorMessage);
+};
+
+promise1
+  .then(onSuccess)
+  .catch(onError);
+
+promise2
+  .then(onSuccess)
+  .catch(onError);
```

Comment:
The promise handling logic for `promise2` is also correct, as it rejects after a timeout and calls `onError`. However, since `promise2` only rejects, the `then` block will never be executed. Consider revising the logic if you intend to handle a successful resolution for `promise2`.